lib/js/components/Chip/Chip.consts.ts

```diff
@@ -24,3 +28,5 @@ export const CHIP_RADIUSES = {
 	CAPSULE: 'capsule',
 	ROUNDED: 'rounded',
 } as const;
+
+export type ChipRadius = keyof typeof CHIP_RADIUSES;
```

Comment:
Those seems to be incorect types as those are returning keys `CAPSULE` | `ROUNDED` and instead we should use `Value` helper `Value<typeof CHIP_RADIUSES>` to return values of const